client/modules/IDE/reducers/sorting.js

```diff
@@ -0,0 +1,12 @@
+import * as ActionTypes from '../../../constants';
+
+const sorting = (state = { orderBy: 'createdAt', order: 'Dsc' }, action) => {
```

Comment:
i'm not sure if it makes sense to make sorting into its own reducer. i would put it in the `ide` reducer, but i could also be convinced that it's too big and needs to be broken up.

Reply:
I feel adding sorting to anything needs a new reducer , I can change accordingly.